data/json/professions.json

```diff
@@ -7273,6 +7273,27 @@
       "female": { "entries": [ { "item": "panties" }, { "item": "bra" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "muckracker",
+    "name": "Muckracker",
+    "description": "You've covered it all before, you know? War, Corruption, the bizzare and more, but even a pulitzer couldn't prepare you for this scoop.",
+    "proficiencies": [ "prof_lockpicking" ],
+    "points": 3,
+    "skills": [ { "level": 2, "name": "pistol" }, { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": {
+        "items": [ "jacket_leather", "dress_shirt", "dress_shoes", "tie_necktie" ],
```

Comment:
No press ballistic vest?

Reply:
i don't know, i imagine a muckraker to be more low-down, you know? Not trying to get much attention to themselves.